### DISCUSSION TITLE
kill_states service

### DISCUSSION THREAD

**Discussion by g-off-white** [travisghansen/hass-pfsense#151] [Q&A]
Hi I want to use kill_states to destroy all existing connections to my daughters ipad when she is naughty in conjunction with a rule to block new ones.  However when I call it with the IP address of the device it does not seem to stop them al the time.

service: pfsense.kill_states
data:
  entity_id: binary_sensor.pfsense_localdomain_pending_notices_present
  source: "10.1.1.25"

When i check the states list they are still there. 

Am I doing it right and is there any way to check the command is going through?

**Comment by guilhermeoq** [comment]
Do you have IPv6 enabled by your ISP? Perhaps her connections are going out to the Internet via IPv6. And the problem with just using kill states is that new states will be created by apps on the iPad. Try leaving the device's IPv4 and IPv6 addresses as static in the DHCP settings. Then create a specific rule on the LAN to block connections from these IPs. You will enable/disable this rule whenever you want.

**Comment by g-off-white** [comment]
I am only using v4 and have static addresses. I use firewall rules to block new connections and use the kill states to break existing ones.

The issue I have is that it does not work all the time.

Regards,

Geoff white
-------------------
Disclaimer: I am blaming any typos in this email on my phone.
Please do not mistake my brevity as rudeness; my rudeness is unmistakable.

On 12 Mar 2023, at 7:17 am, Guilherme Quintino ***@***.***> wrote:

﻿

Do you have IPv6 enabled by your ISP? Perhaps her connections are going out to the Internet via IPv6. And the problem with just using kill states is that new states will be created by apps on the iPad. Try leaving the device's IPv4 and IPv6 addresses as static in the DHCP settings. Then create a specific rule on the LAN to block connections from these IPs. You will enable/disable this rule whenever you want.

**Comment by jasongabler** [comment]
Any progress on this? I'm in the same boat.
